### PR TITLE
Add action input for scala-cli version, default to latest

### PR DESCRIPTION
**Pull request by ptrdom**
Add action input for scala-cli version, default to latest.

### CODE REVIEW

File: src/main.ts
Comment (lwronski) [inline src/main.ts:101]:
```suggestion
      apps.push(`scala-cli${scalaCLIVersionInput ? `:${scalaCLIVersionInput}` : ':${scalaCLIVersion}'}`)
```

**Review by lwronski — COMMENTED** [review]
Hi @ptrdom, thanks for contribution. 

IMO we shouldn't use the latest version of scala-cli by default. The user should be aware that there is a new version of scala-cli, test it and then merge it into the main branch. Without testing, it can generate unexpected behavior on CI

To keep the scala-cli-setup up to date, ScalaCli core team prepares a new task, to open PR with updating scala-cli version in scala-cli-setup. After release, this PR will be merged and created new tag. It will be something similar to the [brew formula ](https://github.com/VirtusLab/scala-cli/blob/44dc360375b46c21bca05027785c8ddaf0ce0327/build.sc#L1154).

**Comment by ptrdom** [comment]
@lwronski All good, makes sense. I thought that defaulting to latest would not be a bad idea with the ability to set a specific version - developers should be quite used to getting the latest version with default installations.

**Comment by lwronski** [comment]
I think, you should also updated [action.yml](https://github.com/VirtusLab/scala-cli-setup/blob/78483860002f6114ed87aca41779f54dd7370377/action.yml#L7) and specify new input parameter. 

> developers should be quite used to getting the latest version with default installations

Right, but in some cases, it can throw an unexpected error, so it is better to test the newer version of `scala-cli-setup` in PR and then merge after CI is passed.

Instead of the default behavior, maybe we should add support for the `scala-cli-version` which accepts value `latest`. In this approach,  users have to specify explicitly using the latest `version` of scala-cli.

**Comment by ptrdom** [comment]
> I think, you should also updated [action.yml](https://github.com/VirtusLab/scala-cli-setup/blob/78483860002f6114ed87aca41779f54dd7370377/action.yml#L7) and specify new input parameter.

Will do.

> > developers should be quite used to getting the latest version with default installations
> 
> Right, but in some cases, it can throw an unexpected error, so it is better to test the newer version of `scala-cli-setup` in PR and then merge after CI is passed.

Totally agree.

> Instead of the default behavior, maybe we should add support for the `scala-cli-version` which accepts value `latest`. In this approach, users have to specify explicitly using the latest `version` of scala-cli.

💯

**Comment by lwronski** [comment]
Hi @ptrdom, thanks again for contribution. Approved

**Comment by lwronski** [comment]
Hi @ptrdom, thanks again for contribution. Approved

**Review by lwronski — APPROVED** [review]

**Pull request merged**
